tools/build_config/Jenkinsfile

```
@@ -294,14 +293,13 @@ pipeline {
     always {
       // archive CTest results file and static analysis report
       archiveArtifacts(
-        artifacts: 'build/Testing/**/*.xml,**/cppcheck.xml',
+        artifacts: 'build/Testing/**/*.xml,**/cppcheck.xml,**/mlint.json',
         fingerprint: true
       )
 
-      publishCppcheck (
-        displayAllErrors: true,
-        allowNoReport: true,
-        pattern: '**/cppcheck.xml'
+      recordIssues(tools:
+                   [cppCheck(pattern: '**/cppcheck.xml', reportEncoding: 'UTF-8'),
+                    issues(id: 'mlint', name: 'MLint', pattern: '**/mlint.json')]
```

```suggestion
      recordIssues(
        tools: [
          cppCheck(pattern: '**/cppcheck.xml', reportEncoding: 'UTF-8'),
          issues(id: 'mlint', name: 'MLint', pattern: '**/mlint.json')
        ]
```

would be more consistent with the surrounding formatting